### PR TITLE
Fixed additional inconsistencies for english labels

### PR DESCRIPTION
**Pull request by clementchong**
With reference to https://github.com/running-elephant/datart/issues/1250

Found few more inconsistencies in the english translation file. Please help review and approve PR if ok to proceed. Thank you.

### CODE REVIEW

File: frontend/src/locales/en/translation.json [hunk 1]
```diff
@@ -976,26 +991,27 @@
         "image": "Image",
         "richText": "Rich Text",
         "timer": "Timer",
-        "iframe": "iframe",
+        "iframe": "iFrame",
```
Comment (Cuiyansong) [inline frontend/src/locales/en/translation.json:994]: From end user point, this world may be just `frame` is more reasonable @xieliuduo @scottsut , but it is ok for this commit. If we decide to use  a more "reasonable" world, we would try to fix it.

File: frontend/src/locales/en/translation.json [hunk 2]
```diff
@@ -34,7 +34,7 @@
     "validation": {
       "required": " is required",
       "invalidPassword": "Password must be 6-20 characters",
-      "passwordNotMatch": "Password and confirm password does not match"
+      "passwordNotMatch": "Password and confirm password do not match"
```
Comment (Cuiyansong) [inline frontend/src/locales/en/translation.json:37]: It is very nice!

**Review by scottsut — APPROVED** [review]

**Comment by scottsut** [comment]
Thanks for the contribution!

File: frontend/src/locales/en/translation.json [hunk 3]
```diff
@@ -981,8 +989,8 @@
       "widgetType": "Widget Type",
       "type": {
         "chart": "Chart",
-        "widgetChart": "privateChart",
-        "dataChart": "publicChart",
+        "widgetChart": "Private Chart",
```
Comment (Cuiyansong) [inline frontend/src/locales/en/translation.json:992]: 由于这段更改导致提交上来的代码测试挂了，也就是说你本地没有提前进行 `npm run prepare` 或者绕过了失败的测试，而继续提交上来，希望下次要注意。

Comment (clementchong) [reply frontend/src/locales/en/translation.json:992]: Sorry sorry, are these referenced in some code logic and cannot be changed? I thought these are just labels for display.

Would "mvn clean package" catch these?

Comment (Cuiyansong) [reply frontend/src/locales/en/translation.json:992]: Don't worry, it is not too bad. It is also good for us to setup a github action which would auto run test for each PR. Also, you can trigger the test by run `npm run test:ci` manually.

Comment (clementchong) [reply frontend/src/locales/en/translation.json:992]: Ok noted, I would try it out. Do I need submit a PR to fix this?

Comment (Cuiyansong) [reply frontend/src/locales/en/translation.json:992]: No, the other team member had fixed it.

Comment (clementchong) [reply frontend/src/locales/en/translation.json:992]: Ok noted, thank you for highlighting this. Sorry for the trouble again!

**Pull request merged**